distributed/scheduler.py

```diff
@@ -1537,6 +1611,19 @@ def _clear_task_state(self):
         ]:
             collection.clear()
 
+    @property
+    def total_occupancy(self):
+        res = 0
+        for group_name, count in self.task_groups_count_global.items():
+            # TODO: Kick out empty counters
+            # TODO: Deal with unknown tasks better
+            if count > 0:
+                duration = self.task_groups[group_name].prefix.duration_average
+                if duration < 0:
+                    duration = 0.5
+                res += duration * count
+        return res + self._network_occ_global
```

Comment:
In an initial draft I was hoping we'd be calling this rarely enough to simply be able to sum the occupancy of all workers. However, this is called to often and we need to apply the same kind of logic for the total as for the local occ